x/crosschain/keeper/grpc_query_inbound_tracker_test.go

```diff
@@ -48,3 +48,36 @@ func TestKeeper_InboundTrackerAll(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, 2, len(res.InboundTracker))
 }
+
+func TestKeeper_InboundTracker(t *testing.T) {
+	t.Run("successfully get inbound tracker", func(t *testing.T) {
+		k, ctx, _, _ := keepertest.CrosschainKeeper(t)
+		hash := sample.Hash().Hex()
+		chainID := chains.GoerliLocalnet.ChainId
+		k.SetInboundTracker(ctx, types.InboundTracker{
+			ChainId:  chainID,
+			TxHash:   hash,
+			CoinType: coin.CoinType_Gas,
+		})
+
+		res, err := k.InboundTracker(ctx, &types.QueryInboundTrackerRequest{
+			ChainId: chainID,
+			TxHash:  hash,
```

Comment:
let's check that hash matches

Reply:
That is part of the key; If the value is fetched, it should mean that the hash matches; is there an edge case you are referring to here?